homeassistant/components/blink/strings.json

```diff
@@ -40,12 +40,21 @@
   "entity": {
     "sensor": {
       "wifi_rssi": {
-        "name": "Wi-Fi RSSI"
+        "name": "WiFi RSSI"
```

Comment:
We use some kind of Microsoft naming guideline and that states that it's Wi-Fi

Reply:
I thought I changed it back. Sorry.